CHANGES.md

```diff
@@ -30,6 +30,9 @@
 
 - `--diff` correctly indicates when a file doesn't end in a newline (#1662)
 
+- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules.
+  Works very alike to flake8's `--stdin-display-name` (#1780)
```

Comment:
```suggestion
- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules (#1780)
```
The sentence about flake8 is more suited for documentation, not a changelog.

Reply:
Sure, I just copied was was in there previously